Source/App/EbAppConfig.c

```diff
@@ -311,7 +311,7 @@ void eb_config_ctor(EbConfig *config_ptr)
 
     config_ptr->enc_mode                                         = 9;
     config_ptr->intra_period                                     = -2;
-    config_ptr->intra_refresh_type                               = 1;
+    config_ptr->intra_refresh_type                               = 2;
```

Comment:
why is the change here? should we also update default value in user guide?

Reply:
The default value for intra refresh seems a bit inconsistent among the encoders.  
In AV1 the default is CRA.
In HEVC the user guide and cfg file show that CRA is the default, however in EbAppConfg the default is set to IDR

Seems like the problem in VP9 is resolved when the intra refresh is set to IDR, but not sure if there's another problem (perhaps with references) that fix would be masking.

Reply:
Hi @intelmark , please refer to the new change for the default value in HEVC PR #[435](https://github.com/OpenVisualCloud/SVT-HEVC/pull/435). Thanks!